impl/src/planning.rs

```diff
@@ -291,23 +292,13 @@ mod tests {
 
   #[test]
   fn test_plan_build_produces_aliased_dependencies() {
-    let toml_file = indoc! { r#"
-    [package]
-    name = "build_produces_aliased_dependencies"
-    version = "0.1.0"
-
-    [lib]
-    path = "not_a_file.rs"
-
-    [dependencies]
-    actix-web = "2.0.0"
```

Comment:
@acmcarther what would be a smaller replacement for these crates?

Reply:
Are you looking for any cargo crate to use that has few dependencies?

https://crates.io/crates/cfg-if only depends on core, so should result in a small dependency tree.

Reply:
I don't think I quite understand this test so yeah, if there's a replacement crate you can give me, I can wire it up and it'd help me understand what this is for.

Reply:
I'm not sure that crate is usable for this test

Reply:
looks like this was added in https://github.com/google/cargo-raze/pull/162

Reply:
Is this comment obsolete?

Reply:
I don't think it is but if the format of committed metadata json dumps is an acceptable workaround for network access, then I'd rather open an issue to update the tests (which both before and after this change, there's work to do. Even literal `TODO`s from times past). If your concern is lines of code I can unformat the files to reduce them to a single line. But to me the formatted dumps are better.

Reply:
Formatted dumps are fine I think.

I don't totally follow your comment about there being more work to do, but filing a bug SGTM.

Reply:
https://github.com/google/cargo-raze/blob/master/impl/src/planning.rs#L806-L811 are these not still relevant? That's what I was referring to as well as other clarity cleanups like this particular test. Its not clear to me what specifically it's testing and I think that could be improved. There's also some naming inconsistencies across all tests in that some are prefixed with `test_` and others are not:
```
test planning::license::tests::empty_license_is_restricted ... ok
...
test planning::tests::test_plan_build_minimum_workspace ... ok
```

Things like this. Does that clarify what I was saying a bit more?

Reply:
I made https://github.com/google/cargo-raze/issues/310

Reply:
@acmcarther with that ticket tracking additional work to do for the unit tests, would this conversation be considered resolved?

Reply:
Those TODOs are ancient and can be disregarded.